### PR TITLE
Add release process and changelog.

### PR DESCRIPTION
**Pull request by yswe**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## Releases
+
+**Currently, release tags are created manually from the CLI using the following commands:**
+
+```sh
+git tag <tag>
+git push origin <tag>
+```
+
+### v0.1.1 (Jul 2, 2024)
```
Comment (bowei) [inline RELEASE.md:31]: You should create a CHANGELOG for this information.

This looks like a reasonable description of what should do: https://keepachangelog.com/en/1.1.0/

probably link that here and add to the procedure.

**Comment by bowei** [comment]
Add CHANGELOG etc. otherwise looks good.

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+However, as the project has the v0 major version, so it makes no stability or
+backward compatibility guarantees.
+
+* MINOR: Adding functionality.
+* PATCH: Bug fixes.
+
+## Releases
+
+Please visit the [changelog](CHANGELOG.md) for a detailed changelog.
```
Comment (bowei) [inline RELEASE.md:24]: Add

Every release should update the CHANGELOG.md

Comment (yswe) [reply RELEASE.md:24]: Done.

File: RELEASE.md (hunk 1 above)
Comment (yswe) [reply RELEASE.md]: Done. I also populated it with the major updates done to the repo, so I think it's in a good shape now. Could you review it please?

Comment (yswe) [reply RELEASE.md]: Oh, sorry, wait a bit please, the changes were not pushed for some reason... I'm looking into it.

Comment (yswe) [reply RELEASE.md]: Done.

**Pull request merged**